### PR TITLE
Add ncdc --HEAD

### PR DESCRIPTION
**Pull request by amrav**
- Put the latest tagged version into a 'stable' block.
- Create a 'head' block that points to the ncdc git repo.
- Add dependencies for tip of master: zlib, bzip2 and ncurses. autconf
and automake are also required for building from the git repository.

### CODE REVIEW

File: Library/Formula/ncdc.rb [hunk 1]
```diff
@@ -14,7 +12,24 @@ class Ncdc < Formula
   depends_on 'pkg-config' => :build
   depends_on 'geoip' => :optional
 
+  stable do
+    url 'http://dev.yorhel.nl/download/ncdc-1.19.tar.gz'
+    sha1 '7f478b7daf09202586b40899cc6beabeb0d23178'
+  end
```
Comment (mikemcquaid) [inline Library/Formula/ncdc.rb]: We don't need a `stable do` block unless there's e.g. stable-specific dependencies or patches.

File: Library/Formula/ncdc.rb [hunk 2]
```diff
@@ -14,7 +12,24 @@ class Ncdc < Formula
...
+
+  head do
+    url "git://g.blicky.net/ncdc.git"
+
+    depends_on 'autoconf' => :build
+    depends_on 'automake' => :build
+    depends_on 'zlib'
+    depends_on 'bzip2'
+    depends_on 'ncurses'
```
Comment (mikemcquaid) [inline Library/Formula/ncdc.rb]: The three above are all provided by OS X.

**Comment by amrav** [comment]
@mikemcquaid I've made the changes you suggested and force pushed a new commit. 

I'm not sure why the automated build is failing on Yosemite. It seems like the gnutls install is failing?

**Comment by mikemcquaid** [comment]
Not sure what's going on there, hmm.

**Comment by amrav** [comment]
The homebrew install succeeds on my computer, but when I manually check out ncdc and follow the same steps as in the formula, I get:

```bash
$ ./configure
[...]
checking for GNUTLS... no
configure: error: Package requirements (gnutls >= 2.4) were not met:

Package 'zlib', required by 'gnutls', not found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables GNUTLS_CFLAGS
and GNUTLS_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.
```

How does homebrew provide pkg-config the zlib package?

**Comment by jacknagel** [comment]
Pulled as 5662d4abb7c18458193f011defaf287e4fa24974

**Pull request closed without merging**